core/common/src/main/java/alluxio/client/file/URIStatus.java

```diff
@@ -236,6 +236,13 @@ public boolean isPinned() {
     return mInfo.isPinned();
   }
 
+  /**
+   * @return the pinned location list
+   */
+  public List<String> getPinnedLocation() {
```

Comment:
Should this be plural? Also, should we use a `Set` instead of a list, or do we expect use cases for duplicated locations?